.github/workflows/rector.yml

```
@@ -8,13 +8,20 @@ on:
         default: composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
         required: false
         type: string
+      download-config:
+        description: Download Rector configuration file
+        default: true
```

```suggestion
        default: false
```

`True` as default value will be incorrect work in existing packages.

---

done.